.changeset/real-cars-mate.md

```diff
@@ -0,0 +1,10 @@
+---
+'@swisspost/design-system-styles': major
+---
+
+Updated the file structure for the `post-footer` component global styles:
+
+- dist/components/globals/post-footer.scss -> dist/components/footer/index.scss
+- dist/components/globals/post-footer.css -> dist/components/footer/index.css
```

Comment:
```suggestion
- dist/components/globals/post-footer.scss -> dist/components/footer/index.[s]css
```

Reply:
This feels like a custom notation imo.
If we change this, wouldn't it be better to write it like so:
```md
- dist/components/globals/post-footer.(css|scss) -> dist/components/footer/index.(css|scss)
```
???